spec/services/linked_data/mapper/graph_ldpath_mapper_service_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'spec_helper'
+
+RSpec.describe Qa::LinkedData::Mapper::GraphLdpathMapperService do
+  let(:graph) { Qa::LinkedData::GraphService.load_graph(url: 'http://local.data') }
```

Comment:
These are the same tests run for GraphPredicateMapperService to be sure both services perform the same.